test/NetTopologySuite.Samples.Console/Tests/Operation/Linemerge/LineSequencerTest.cs

```diff
@@ -200,6 +203,27 @@ public void BadLineSequence()
             RunIsSequenced(wkt, false);
         }
 
+        [Test]
```

Comment:
Can this be moved to `NetTopologySuite.Tests.NUnit.Operation.LineMerge.LineMergerTest`?

Reply:
Yes I'll move it there